core/src/scene/stops.cpp

```diff
@@ -90,10 +90,13 @@ auto Stops::Sizes(const YAML::Node& _node, uint8_t _units) -> Stops {
     float lastKey = 0;
 
     auto constructFrame = [&](const auto& _frameNode, StyleParam::ValueUnitPair& _result) -> bool {
-        if (StyleParam::parseSizeUnitPair(_frameNode.Scalar(), 0, _result)) {
-            if ( !(_units & _result.unit) ) {
+        if (StyleParam::parseSizeUnitPair(_frameNode.Scalar(), _result)) {
+            if ( !_units.contains(_result.unit) ) {
                 LOGW("Size StyleParam can only take in pixel, %% or auto values in: %s", Dump(_node).c_str());
-                return false;
+                _result.unit = Unit::pixel;
```

Comment:
Does it make sense to correct an erroneous unit value provided by the user? Maybe just have the warning like before.

Reply:
Tangram JS seems to use the default unit if the suffix is not recognized. I'll keep this the same.